src/script/descriptor.cpp

```diff
@@ -843,6 +907,15 @@ class MultiADescriptor final : public DescriptorImpl
 public:
     MultiADescriptor(int threshold, std::vector<std::unique_ptr<PubkeyProvider>> providers, bool sorted = false) : DescriptorImpl(std::move(providers), sorted ? "sortedmulti_a" : "multi_a"), m_threshold(threshold), m_sorted(sorted) {}
     bool IsSingleType() const final { return true; }
+
+    std::optional<int64_t> ScriptSize() const override {
+        const auto n_keys = m_pubkey_args.size();
+        return (1 + 32 + 1) * n_keys + BuildScript(m_threshold).size() + 1;
+    }
+
+    std::optional<int64_t> MaxSatSize(bool use_max_sig) const override {
+        return (1 + 65) * m_threshold + (m_pubkey_args.size() - m_threshold);
+    }
```

Comment:
In 9e8df2de1750b823492c18a924fefbd7d474819e "descriptor: introduce a method to get the satisfaction size"

Is `MultiADescriptor` supposed to be missing a `MaxSatisfactionWeight`?

Reply:
Yes. It would never be used as `MaxSatisfactionWeight` is only called on top-level descriptors.